backup: refactor some docs for easier understanding (#12338)
pingcap/docs: br/backup-and-restore-storages.md

TiDB supports storing backup data to Amazon S3, Google Cloud Storage (GCS), Azure Blob Storage, and NFS. Specifically, you can specify the URL of backup storage in the `--storage` or `-s` parameter of `br` commands. This document introduces the [URL format](#url-format) and [authentication](#authentication) of different external storage services, and [server-side encryption](#server-side-encryption).

## Send credentials to TiKV

> 

Reviewer: 这里中文里还有 命令行参数 的表格

Author: 只有一个元素，感觉不需要用表格来呈现，到时候会再改下中文。

Author: Fixed in [619c6d2](https://github.com/pingcap/docs/pull/12338/commits/619c6d280b0d96f4e3ce61b65d508f8543beb099)

By default, BR sends a credential to each TiKV node when using Amazon S3, GCS, or Azure Blob Storage as the storage system. This behavior simplifies the configuration and is controlled by the parameter `--send-credentials-to-tikv`. The default value of this parameter is `true`.

> Note that this operation is not suitable for cloud environments. If you use IAM Role authorization, each node has its own role and permissions. In this case, you need to configure `--send-credentials-to-tikv=false` (or `-c=0` in short) to disable sending credentials:

Reviewer: ```suggestion
Note that this operation is not applicable to cloud environments. If you use IAM Role authorization, each node has its own role and permissions. In this case, you need to configure `--send-credentials-to-tikv=false` (or `-c=0` in short) to disable sending credentials:
```

```bash
./br backup full -c=0 -u pd-service:2379 --storage 's3://bucket-name/prefix'
```

> If you backup or restore data using the [`BACKUP`](/sql-statements/sql-statement-backup.md) and [`RESTORE`](/sql-statements/sql-statement-restore.md) statements, you can add the `SEND_CREDENTIALS_TO_TIKV = FALSE` option:

Reviewer: ```suggestion
If you back up or restore data using the [`BACKUP`](/sql-statements/sql-statement-backup.md) and [`RESTORE`](/sql-statements/sql-statement-restore.md) statements, you can add the `SEND_CREDENTIALS_TO_TIKV = FALSE` option:
```

TiDB supports storing backup data to Amazon S3, Google Cloud Storage (GCS), Azure Blob Storage, and NFS. Specifically, you can specify the URL of backup storage in the `--storage` or `-s` parameter of `br` commands. This document introduces the [URL format](#url-format) and [authentication](#authentication) of different external storage services, and [server-side encryption](#server-side-encryption).

## Send credentials to TiKV

| CLI parameter | Description | Default value
|:----------|:-------|:-------|

> | `--send-credentials-to-tikv` | Controls whether to send credentials obtained by BR to TiKV | `true`|

Author: ```suggestion
| `--send-credentials-to-tikv` | Controls whether to send credentials obtained by BR to TiKV. | `true`|
```

> By default, BR sends a credential to each TiKV node when using Amazon S3, GCS, or Azure Blob Storage as the storage system. This behavior simplifies the configuration and is controlled by the parameter `--send-credentials-to-tikv`.

Reviewer: ```suggestion
By default, BR sends a credential to each TiKV node when using Amazon S3, GCS, or Azure Blob Storage as the storage system. This behavior simplifies the configuration and is controlled by the parameter `--send-credentials-to-tikv`(or `-c` in short).
```
